### ISSUE TITLE
Grouped radio buttons in Settings dialog not labeled

### ISSUE THREAD

**Issue by becka11y** [readium/readium-js-viewer#166]
The heading for the grouped radio buttons in the settings dialog is not announced by the screen readers.
We just need to add an aria-labelledby pointing to the h5 containing the label.  It looks like this was done at one time but then commented out.  

It does make the radios a bit verbose as the group title is announced for each radio button but at least the label is announced.
